iOS/BugsnagTests/Swift Tests/BugsnagSwiftTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  BugsnagSwiftTests.swift
+//  Tests
+//
+//  Created by Robin Macharg on 05/02/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+//  Swift unit tests of global Bugsnag behaviour
+
+import XCTest
+
+class BugsnagSwiftTests: XCTestCase {
+
+    /**
+     * Confirm that the addMetadata() method is exposed to Swift correctly
+     */
+    func testAddMetadataToSectionIsExposedToSwiftCorrectly() {
+        do {
+            if let configuration = try BugsnagConfiguration(DUMMY_APIKEY_32CHAR_1) {
+                Bugsnag.start(with: configuration)
+                Bugsnag.addMetadata("mySection1", key: "myKey1", value: "myValue1")
+                
+                let exception1 = NSException(name: NSExceptionName(rawValue: "exception1"), reason: "reason1", userInfo: nil)
+                
+                Bugsnag.notify(exception1) { (event) in
+                    // Arbitrary test, replicating the ObjC one
+                    let value = (event.metadata["mySection1"] as? [String : Any])?["myKey1"] as? String
```

Comment:
`event.metadata` should be replaced by `[event getMetadata]` and all the other associated methods - for the purposes of this review I'm assuming that this will be implemented in another PR.

Reply:
👍